src/main/java/org/folio/rest/impl/PutOrderLineByIdHelper.java

```diff
@@ -162,9 +163,14 @@ public boolean isUpdateSuccessful() {
     // Check if any item should be created
     int expectedItemsQuantity = calculateInventoryItemsQuantity(compPOL);
     if (expectedItemsQuantity == 0) {
-      logger.debug("PO Line with '{}' id does not require inventory updates", compPOL.getId());
-      return completedFuture(null);
+    	// Create pieces if items does not exists
+    	return createPieces(compPOL, null)
+    	.thenRun(() -> {
```

Comment:
The result of the `updateInventory(CompositePoLine)` method is `CompletableFuture<Void>`. I believe `thenRun` can be replaced by `thenAccept` in case you would like info log message once all pieces created successfully. In case you prefer to use `thenRun`, `completedFuture(null)` is not required there...

Reply:
I have removed `completedFuture(null)`since `thenRun` does not return anything at the end of the chain